lowell-dashboard/app/models.py

```diff
@@ -150,6 +151,31 @@ def get_username(self):
     def __repr__(self):
         return self.get_full_name()
 
+    '''
+    A getter function for the class id property
+    Args:
+        None
+    Returns:
+        list: a list of the class ids that the student has
+    Use case:
+    print(CustomUser.class_ids)
+    ['12', '27']
+    '''
+    @property
+    def class_ids(self):
+        return [float(x) for x in self._class_ids.split(';')]
+    '''
```

Comment:
just to confirm we are not saving as list just a string with items separated by ";" semicolons right.

Reply:
Yes database tables don't exactly have arrays or list as column types. So we just store them as a String and we parse it later when we need to retrieve the data.